components/rhs_card/rhs_card.tsx

```diff
@@ -1,59 +1,71 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import PropTypes from 'prop-types';
 import React from 'react';
 import Scrollbars from 'react-custom-scrollbars';
 import {FormattedMessage} from 'react-intl';
 import {Link} from 'react-router-dom';
+import {Post} from 'mattermost-redux/types/posts';
+import {Channel} from 'mattermost-redux/types/channels';
 
 import DelayedAction from 'utils/delayed_action';
-import Constants, {RHSStates} from 'utils/constants';
+import Constants, {RHSStates, ValueOf} from 'utils/constants';
 import * as Utils from 'utils/utils.jsx';
 import RhsCardHeader from 'components/rhs_card_header';
 import Markdown from 'components/markdown';
 import UserProfile from 'components/user_profile';
 import PostProfilePicture from 'components/post_profile_picture';
 import * as GlobalActions from 'actions/global_actions.jsx';
+import {PluginsState} from '../../types/store/plugins';
+
+interface Props {
+    selected?: Post;
+    pluginPostCardTypes: PluginsState['postCardTypes'];
+    previousRhsState?: ValueOf<typeof RHSStates>;
+    enablePostUsernameOverride?: boolean;
+    teamUrl: string;
+    channel?: Channel;
```

Comment:
This isn't used in the component anywhere and should be removed.

Reply:
It was present in tests, so I decided to include it here. I will remove it